app/views/registration.html

```diff
@@ -148,7 +153,9 @@ <h2 class="page-title border details-heading" translate>
         </div>
         <div class="col-sm-9 col-xs-6">
           <a
-            ng-href="{{registerMode}}/{{conference.id}}/page/{{conference.registrationPages[0].id}}?reg={{currentRegistration.registrants[0].id}}"
+            ng-href="{{ registerMode }}/{{ conference.id }}/page/{{
+              conference.registrationPages[0].id
+            }}?reg={{ currentRegistration.registrants[0].id }}"
```

Comment:
Are you changing the formatting on this HTML or did Prettier somehow not run on these files initially? These changes are fine.

Reply:
Prettier seems to not have ran on some files here and there which is strange.